### PR TITLE
Feat/use resizin hook

### PR DESCRIPTION
**Pull request by horaklukas**
* reorganize sources in `resizin-react` package, move them into subfolders by category
* add `useImage` hook for generating image url
* export `Options` interface from `resizin` package
* fix vulnerability warning about `serialize-javascript` pkg by upgrading `react-styleguidist` and `webpack`

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ This is a guide for those responsible for publishing new package versions.
     yarn release
     ```
 
-    * Choose correct version for each package in version and confirm
+    * Choose correct version of each chaned package and confirm
```
Comment (jancama2) [inline CONTRIBUTION.md]: `changed`

File: packages/react-resizin/README.md [hunk 2]
```diff
@@ -74,6 +75,43 @@ Render image as a standard `img` tag (unless you provide children func, see belo
...
+```
+
+and can be used as
+
+```jsx
+import { ResizinProvider, useImage } from 'react-resizin';
+
+const MyContent = () => {
+    const url = useImage('walle', {
+        filter: 'blur',
```
Comment (jancama2) [inline packages/react-resizin/README.md]: :heart:

File: packages/react-resizin/src/hooks/useImage.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Options } from 'resizin';
+
+import ResizinContext from '../components/ResizinContext';
+
+const useImage = (imageId: string, options: Options): string => {
+    const buildImageUrl = React.useContext(ResizinContext);
```
Comment (jancama2) [inline packages/react-resizin/src/hooks/useImage.ts]: It is usual to check if the component or the hook is used out of the provider of the context and if it is, then to log some error or warnin.g

File: CONTRIBUTION.md (hunk 1 above)
Comment (horaklukas) [reply CONTRIBUTION.md]: ✅

File: packages/react-resizin/src/hooks/useImage.ts (hunk 3 above)
Comment (horaklukas) [reply packages/react-resizin/src/hooks/useImage.ts]: Great idea, adde (inspired by react-redux) along with hook tests 👍

**Pull request merged**